### ISSUE TITLE
`style.shape` property has on effect on `FilledTonalButton` and `FilledButton` in Flet 0.25.2

### ISSUE THREAD

**Issue by Klu1d** [flet-dev/flet#4568]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

Hello,

After upgrading to Flet 0.25.2, I encountered two issues that seem to be related, so I am reporting them together:
Issue 1: `FilledTonalButton` and `FilledButton` not displaying

When running a statically packed Flet application (using flet publish), `FilledTonalButton` and `FilledButton` do not render. This behavior was not observed in version 0.24.1.
Issue 2: Rounded corners for buttons not working

In version 0.25.2, the shape property for buttons (e.g., `ft.RoundedRectangleBorder`) no longer works as expected for  `FilledTonalButton` and `FilledButton`. This issue also appears to have been introduced with the update.

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    page.add(
        ft.Column(
            expand=True,
            controls=[
                ft.Column(
                    expand=1,
                    controls=[
                        ft.Text('Issue 1: Problem with displaying FilledTonalButton and FilledButton on a static Flet.'),
                        ft.ElevatedButton('ElevatedButton'),
                        ft.OutlinedButton('OutlinedButton'),
                        ft.FilledButton('FilledButton'),
                        ft.FilledTonalButton('FilledTonalButton'),
                    ]
                ),
                ft.Column(
                    expand=1,
                    controls=[
                        ft.Text('Issue 2: Problem with button rounding on a static Flet.'),
                        ft.ElevatedButton(
                            'ElevatedButton',
                            style=ft.ButtonStyle(shape=ft.RoundedRectangleBorder(radius=5))
                        ),
                        ft.OutlinedButton(
                            'OutlinedButton',
                            style=ft.ButtonStyle(shape=ft.RoundedRectangleBorder(radius=5))
                        ),
                        ft.FilledButton(
                            'FilledButton',
                            style=ft.ButtonStyle(shape=ft.RoundedRectangleBorder(radius=5))
                        ),
                        ft.FilledTonalButton(
                            'FilledTonalButton',
                            style=ft.ButtonStyle(shape=ft.RoundedRectangleBorder(radius=5))
                        ),
                    ]
                ),
            ]
        )
    )

ft.app(main)

```

### Steps to Reproduce:  
1. Save the code above as `main.py`.  
2. Use the following commands to publish the app and serve it locally:  
   ```
   flet publish main.py --assets assets --app-name FletIsAwesome --app-description FletIsAwesome
   python.exe -m http.server 8080 --bind 127.0.0.1 --directory dist
   ```
3. Access the app in the browser (e.g., http://127.0.0.1:8080).  
4. Observe the following issues:  
   - Buttons `FilledButton` and `FilledTonalButton` do not appear.  
   - Rounded corners do not apply to these buttons.  

### Expected behavior

I expect that when packaging a static application, it should work visually the same as a desktop application. At least in terms of appearance, as shown in the video.

### Screenshots / Videos

**flet run:**
Expected behavior

https://github.com/user-attachments/assets/9bcdae49-e610-4b97-85cb-2e783dddde77

**flet publish:**
Result

https://github.com/user-attachments/assets/f797ad85-2292-4078-acdb-339aa11065d4

### Operating System

Windows

### Operating system details

Windows 11 24H2

### Flet version

0.25.2

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

**Comment by FeodorFitsner** [comment]
It could be browser cache. Try in InPrivate/Incognito mode. I'm getting this:

<img width="1033" alt="image" src="https://github.com/user-attachments/assets/75e8b097-a6c2-41cc-9668-51bcb7f04c09" />

**Comment by Klu1d** [comment]
> It could be browser cache. Try in InPrivate/Incognito mode. I'm getting this:
> <img alt="image" width="1033" src="https://private-user-images.githubusercontent.com/5041459/395800961-75e8b097-a6c2-41cc-9668-51bcb7f04c09.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyMDIyODIsIm5iZiI6MTczNDIwMTk4MiwicGF0aCI6Ii81MDQxNDU5LzM5NTgwMDk2MS03NWU4YjA5Ny1hNmMyLTQxY2MtOTY2OC01MWJjYjdmMDRjMDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTRUMTg0NjIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWIzNTUyZTUwOGU4NWQ2ZmY5ZTE5NjgyY2YwODBkNGRjYWE0MjdmM2E4YzVkZDY5OTM1NjFlNDI1Y2UxN2YxZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.g0Y6GLdsSkMWq1Bzy-FFhyQqEaDOKwIXGNXUigCxsWo">

I think you're right about the cache. However, clearing it completely or opening it in a new window hasn't worked for me so far. I'll try it on another computer.

I see that issue #1 is not present in your screenshot, but issue #2 is there. If you ran my example code, the corner radius should be set to 5. In other words, it should look more rectangular than what you currently have.

**Comment by Klu1d** [comment]
Yes, I was able to achieve the desired result in incognito mode, which means it should also be possible without it. Yes, this is related to caching, and it needs to be cleared.

**Comment by FeodorFitsner** [comment]
I can confirm #2 issue exists and I can reproduce it. Will be looking into that.

**Comment by ndonkoHenri** [comment]
> Issue 2: Rounded corners for buttons not working
In version 0.25.2, the shape property for buttons (e.g., ft.RoundedRectangleBorder) no longer works as expected for FilledTonalButton and FilledButton. This issue also appears to have been introduced with the update.

Fixed in the latest Flet version: `pip install 'flet[all]' --upgrade`
